crowbar_framework/app/assets/javascripts/barclamps/crowbar/application.js

```diff
@@ -16,6 +16,26 @@
  */
 
 jQuery(document).ready(function($) {
+  function nodeStatus() {
```

Comment:
The old generic plugin did a complete page reload, this is not what we want for this simple status change.